src/app/events/[eventId]/digitalWaiver/1/page.tsx

```diff
@@ -7,25 +7,25 @@ import {
   Textarea,
   Link as ChakraLink,
 } from "@chakra-ui/react";
-import styles from './page.module.css'
-import beaverLogo from '/docs/images/beaver-logo.svg'
-import Image from 'next/image'
+import styles from "./page.module.css";
+import beaverLogo from "/docs/images/beaver-logo.svg";
+import Image from "next/image";
 import NextLink from "next/link";
-
+import { wrap } from "module";
```

Comment:
whats this being used for?

Reply:
it's unused I think my vs code must have auto-added it when I used autocomplete for one of the CSS styling options that I ended up removing.  I'll remove that line right now